### PR TITLE
Mapper

### PR DESCRIPTION
**Pull request by brenpardal**
### **WHAT'S NEW**
**This update includes:**
* Added mapper to the object data

#### *Log:*
mapper

#### *Package tree:*
package_tree

### CODE REVIEW

File: app/src/main/java/com/weatherapp/data/service/WeatherService.kt [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+    private val mapper = WeatherMapper()
 
-    fun getWeather(id: String): Observable<WeatherResponse> {
+    fun getWeather(id: String): Observable<Weather> {
         return Observable.create { subscriber ->
             val callResponse = api.createService(WeatherApi::class.java).getCurrentWeatherData(id, BuildConfig.WEATHER_API_KEY, UNITS)
             val response = callResponse.execute()
             if (response.isSuccessful) {
-                response.body()?.let { subscriber.onNext(it) }
+                response.body()?.let {
```
Comment (BQuinteros) [inline app/src/main/java/com/weatherapp/data/service/WeatherService.kt:19]: you can put all in the same line or do an enter in the second brace

Comment (brenpardal) [reply app/src/main/java/com/weatherapp/data/service/WeatherService.kt:19]: Solved.

**Review by SartiMau — COMMENTED** [review]
Approved after Brai's comment

**Review by SartiMau — APPROVED** [review]

**Review by BQuinteros — APPROVED** [review]

File: app/src/main/java/com/weatherapp/data/entity/Data.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.weatherapp.data.entity
+
+data class Data(
```
Comment (lucianoeli) [inline app/src/main/java/com/weatherapp/data/entity/Data.kt:3]: can you give more decriptive name?

cause if you add a new service is going to be confuse

**Review by lucianoeli — APPROVED** [review]

**Pull request merged**